modules/mono/glue/GodotSharp/GodotSharp/Core/GodotObject.base.cs

```diff
@@ -174,18 +176,18 @@ public SignalAwaiter ToSignal(GodotObject source, StringName signal)
             return new SignalAwaiter(source, signal, this);
         }
 
-        internal static Type InternalGetClassNativeBase(Type t)
+        internal static Type? InternalGetClassNativeBase(Type? t)
```

Comment:
At first I thought that it'd need to be a nullable input because it's technically passed a `Type?` value in `ScriptManagerBridge.cs`, but after double-checking it shouldn't ever be null at that stage, so that's a safe change to make

The only change inside the method was appending `!` to `t.BaseType` so it doesn't throw warnings against the concatenation shorthand. Unsure if the method warrants refactoring to avoid that sidestep altogether

Reply:
I think annotating this method may require more discussion (see https://github.com/godotengine/godot/pull/82983#discussion_r1350205161).